packages/mobile/ios/Podfile.lock

```diff
@@ -15,83 +20,104 @@ PODS:
     - React-Core (= 0.63.4)
     - React-jsi (= 0.63.4)
     - ReactCommon/turbomodule/core (= 0.63.4)
-  - Firebase/Auth (7.7.0):
+  - Firebase (7.11.0):
+    - Firebase/Core (= 7.11.0)
+  - Firebase/Auth (7.11.0):
```

Comment:
Updating Firebase or other unrelated dependencies should be done outside of this PR.

Reply:
same as above ^

Reply:
Sorry I should have documented this somewhere. I needed the latest firebase libs for Android 10 to work because they changed how FCM works. That is all fine with android. With iOS the segment library is not updated yet so I had to find something in between that works with android and still works on iOS with segment. As soon as segment updates we can go to the latest firebase. Does that make sense?